### PR TITLE
update macOS build steps

### PR DESCRIPTION
**Pull request by fanckush**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -124,52 +124,65 @@ The result of the compilation should be the binaries `hdrmerge.exe` and `hdrmerg
 You have finished.
 
 ## Compilation in macOS
+The first step is to get all the dependencies as well as the sounce code:
```
Comment (heckflosse) [inline INSTALL.md]: s/sounce/source

File: INSTALL.md [hunk 2]
```diff
@@ -124,52 +124,65 @@ The result of the compilation should be the binaries `hdrmerge.exe` and `hdrmerg
...
-9. Make the .dmg
-```
+
+8. Make the .dmg for further distribution (optional)
+``` bash
 sudo hdiutil create -ov -srcfolder ~/hdrmerge/build/install/hdrmerge.app ~/hdrmerge/build/install/HDRMerge.dmg
 ```
 
-
+That was it! You can move the hdrmerge.app to you applicotions folder and start using it!
```
Comment (heckflosse) [inline INSTALL.md]: s/applicotions/applications

**Comment by Beep6581** [comment]
@fanckush @heckflosse 's remarks are marked as "resolved", but the errors are still there.

@Benitoite are you happy with the contents?

**Comment by fanckush** [comment]
@Beep6581 I was not aware of how the code review process works and thought that the changes have been made.
It's fixed now

**Comment by Benitoite** [comment]
@Beep6581  It looks fine... I will have to install `brew` on my 10.12 machine to test it (my 10.14 mac is strictly sans pkg management).  I think we should figure out what catch-all `brew` command will work and include that, the way some projects have a complete, one-shot `apt-get` command for linuxers.

**Comment by Benitoite** [comment]
@fanckush  please see the results of my testing on 10.12 *Sierra* with homebrew
https://github.com/fanckush/hdrmerge/pull/1

**Comment by Benitoite** [comment]
@fanckush once you're happy with my changes (you can make further changes) and merge https://github.com/fanckush/hdrmerge/pull/1 to your patch, it should be

**Comment by fanckush** [comment]
I merged the PR. however I removed the git step; I just installed a VM with fresh Sierra 10.12.6 and Xcode Developer Tools do install git behind the scene 👍

If no one has further modifications, I think this can be merged now

**Review by Benitoite — APPROVED** [review]
Right on! @fanckush

**Pull request merged**